torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -125,13 +125,13 @@ PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   return array.release();
 }
 
-at::Tensor tensor_from_numpy(PyObject* obj) {
+at::Tensor tensor_from_numpy(PyObject* obj, bool copy_numpy) {
```

Comment:
```suggestion
at::Tensor tensor_from_numpy(PyObject* obj, bool copy_numpy=false) {
```

Reply:
> 

It throws an error if the default value is passed in the function implementation. So, false is passed during function declaration only.

Reply:
https://github.com/pytorch/pytorch/pull/47271/commits/37518cd4960374c10c716f50e58f814a25ce327e added the comment for reference in implementation.

Reply:
oops. sorry I meant to comment on the other macro path.